src/components/VirtualTryOn/index.jsx

```
@@ -0,0 +1,610 @@
+          if (e.key === 'Enter' || e.key === ' ') {
+            e.preventDefault();
+            inputRef.current?.click();
+          }
+        }}
+      >
+        <div className={styles.cardContentArea}>
+          <input
+            type="file"
+            ref={inputRef}
+            onChange={(e) => {
+              if (e.target.files && e.target.files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)